tutorials/spa-create-automation/spa-create-automation.md

```diff
@@ -106,59 +106,80 @@ Business project usually need to use parameters and variable at runtime. These v
 
 You will now design the automation in the Automation Editor by dragging-and-dropping activities into the workflow of the automation. Later you will configure the inputs and outputs of each activity. You will need activities to interact with Microsoft Excel application. These activities will open Excel application, open the workbook that contains the sales orders details and map them into a data type that will be created during the design. Last, after extracting and mapping the data, the Excel application will be closed.
 
-1. Double click on the automation **Get Order Details** which navigates to the Design Studio to build your automation.
+1. Select 3 dots next to **Get Order Details**, choose **Open Editor**, which navigates to the Design Studio to build the automation.
 
-    Since excel is used in this automation, you have to open an excel instance. **Open Excel Instance** is a mandatory activity to use when using Excel. It opens an instance of Excel. Once you open an Excel instance, you can use other Excel activities.
+    ![001](009.png)
 
-2. To open **Excel Instance**, do the following:
+    > Since excel is used in this automation, you have to open an excel instance. Open Excel Instance is a mandatory activity to use when using Excel. Once you open an Excel instance, you can use other Excel activities.
```

Comment:
Spelling error (excel).


"Opening an Excel instance is a mandatory activity..."

"Once you have opened an Excel instance..."

Reply:
changes made